src/components/Pagination.vue

```diff
@@ -51,21 +53,15 @@ export default {
     isFirstQuestion(){
       const objectID = this.$route.params.id;
       const pageNum = JSON.parse(objectID);
-      if (pageNum === 0) {
+      if (pageNum === 1) {
         return true;
       }
       return false;
```

Comment:
```suggestion
      return pageNum === 1;
```

Little boolean if/return refactor here